src/linux/Packaging.Linux/install-from-source.sh

```diff
@@ -114,6 +114,14 @@ apt_install() {
     $sudo_cmd apt install $pkg_name -y 2>/dev/null
 }
 
+print_unsupported_distro() {
+    prefix=$1
+    distro=$2
+
+    echo "$prefix: $distro is not officially supported by the GCM Project."
```

Comment:
Lowercase project 'p'?

```suggestion
    echo "$prefix: $distro is not officially supported by the GCM project."
```